### PR TITLE
Change behavior when mission callback results in same mission

### PR DESCRIPTION
**Pull request by tabiosg**
If the user sends a change mission service request but the mission does not actually change, then the program should not waste time trying to create a new video output (important since creating a video output takes a few seconds).

### CODE REVIEW

File: src/esw/cameras/cameras.py [hunk 1]
```diff
@@ -515,31 +518,23 @@ def _update_active_cameras(self) -> None:
         for index, pipeline in enumerate(self._pipelines):
             self._active_cameras[index] = pipeline.device_number
 
-    def _update_all_pipe_video_info(self) -> None:
+    def _update_all_pipe_video_info_and_outputs(self) -> None:
```
Comment (tabiosg) [inline src/esw/cameras/cameras.py:521]: Does this need a better name? Not sure what I can call it

Comment (wilsonchen02) [reply src/esw/cameras/cameras.py:521]: how much more verbose can it be lol

Comment (tabiosg) [reply src/esw/cameras/cameras.py:521]: i meant a shorter name

Comment (tabiosg) [reply src/esw/cameras/cameras.py:521]: ill just merge the pr anyway

**Pull request merged**